guides/source/action_mailer_basics.md

```diff
@@ -841,8 +865,12 @@ instance from your view with [`mailer`][MailHelper#mailer], and accessing the me
 Action Mailer Configuration
 ---------------------------
 
-The following configuration options are best made in one of the environment
-files (environment.rb, production.rb, etc...)
+This section highlights some of the configuration options for Action Mailer. For
+a complete list see the [Configuring Rails
+Applications](configuring.html#configuring-action-mailer) guide.
+
+You can specify these configuration options in environment files such as
+environment.rb, production.rb, etc.
```

Comment:
I don't think we recommend adding configuration to config/environment.rb anymore:
```suggestion
production.rb, etc.
```